docs/_data/mentors/2017.yml

```diff
@@ -36,3 +36,14 @@
     forTravel: false
   groups:
     - "[Nashville PHP](http://nashvillephp.org)"
+
+- name: Kalle Sommer Nielsen
+  location: København, Denmark
+  twitter: https://twitter.com/KalleZ
+  email: kalle@php.net
+  available:
+    forLocal: true
+    forRemote: true
+    forTravel: false
+  groups:
+    - "[Copenhagne PHP](https://www.meetup.com/copenhagenphp/)"
```

Comment:
Hi Kalle! It looks like you have a typo in the link text here ("Copenhagne"). Please correct or let us know if this is intended. Thanks!